### PR TITLE
Feature/lesson information

### LINKED ISSUE #67: LessonDetails component
- [x] Should see the lesson title
- [x] Should see the lesson link
- [x] Should see the lesson statistics

### PR DESCRIPTION
**Pull request by Melizzo**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures
## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 
## What is the current behavior? 
-Lesson preview only shows title

- When a user clicks on a specific title of an assessment, a title, link, average score, and list of questions shows up. 
## What is the new behavior?

- I added in a the new get for average score of a lesson
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

closes #67 
Screen Shot 2020-09-15 at 3 05 13 PM

### CODE REVIEW

File: src/LessonDetails/LessonDetails.js [hunk 1]
```diff
@@ -1,17 +1,42 @@
...
 
-const LessonDetails = ({ lesson }) => {
-return(
-  <section>
-    <h3>{lesson.name}</h3>
-    <p>Scores will be shown here</p>
-  </section>
-)
-}
+const LessonDetails = ({ lesson, lessonAverage }) => {
```
Comment (Billwilke42) [inline src/LessonDetails/LessonDetails.js:5]: Nice work on this!

File: src/LessonDetails/LessonDetails.test.js [hunk 2]
```diff
@@ -7,14 +7,33 @@ import { lesson } from '../mockData/mockData';
 
 const mockLesson = {
   id: 1,
-  name: 'Mocked Lesson'
+  name: 'Mocked Lesson',
```
Comment (Billwilke42) [inline src/LessonDetails/LessonDetails.test.js:10]: great mocks!

**Review by Billwilke42 — APPROVED** [review]
nice job Melissa!

**Review by KyleWong2510 — APPROVED** [review]
Nice Job!

**Review by Billwilke42 — COMMENTED** [review]
Great job

**Pull request merged**